configurations/manager/10GB_dataset.yaml

```diff
@@ -0,0 +1,8 @@
+round_robin: true
+
+prepare_write_cmd: ["cassandra-stress write cl=ONE n=10485760 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=1..10485760"]
+stress_read_cmd: ["cassandra-stress read cl=ONE n=10485760 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native  -rate threads=50 -col 'size=FIXED(64) n=FIXED(16)' -pop seq=1..10485760"]
+
+instance_type_db: 'i4i.2xlarge'
```

Comment:
this overrides db type from `test-cases/manager/manager-no-delta-backup-set-dataset.yaml` which one is valid?

Reply:
Fixed, removed `instance_type_db` from test-cases yaml

Reply:
Ah, okay, test-cases yaml should contain instance_type_db parameter, swapped

Reply:
just to clarify, anything under `test-cases` folder is checked, and expected to be full functioning cases as it, hence all the mandatory field are checked

anything under `configurations` is known to be partial configuration, and not being tested by CI for correctness and missing options.